composer.json

```diff
@@ -19,7 +19,8 @@
         "php" : "^7.2",
         "ext-mongodb": "*",
         "ramsey/uuid": "~3.0|^4",
-        "doctrine/mongodb-odm": "^2.0"
+        "doctrine/mongodb-odm": "^2.0",
+        "doctrine/persistence": "^2.1"
```

Comment:
Because of the dependency to `Doctrine\Persistence\Mapping\MappingException` I added `doctrine/persistence`.